docs/v1/P4Runtime-Spec.mdk

```diff
@@ -3715,6 +3715,22 @@ semantics.
   operation. Any packets with their `multicast_group` metadata in the data plane
   set to the deleted `multicast_group_id` will be dropped.
 
+When reading a multicast group, only `multicast_group_id` is considered. All
+other fields in `MulticastGroupEntry`` are ignored. To perform a *wildcard* read
+on all configured multicast group entries, the client must set the
+`multicast_group_id` field to `MULTICAST_GROUP_ID_WILDCARD`, which is defined in
+p4runtime.proto as an enum member with the value `0xffffffff`:
+
+~ Begin Proto
+enum MulticastGroupIdReserved {
+  // 0xffffffff
+  MULTICAST_GROUP_ID_WILDCARD = -1;
+}
+~ End Proto
+
+This reserved value can never appear in a `Write` RPC. If it does, the server
```

Comment:
I can not appear as a multicast_group_id only, or it can't appear at all?  Same below.

Reply:
I don't think there is a distinction in this case, but I updated my PR to clarify.